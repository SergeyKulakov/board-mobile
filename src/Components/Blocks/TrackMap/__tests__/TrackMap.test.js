import React from 'react'
import { shallow } from 'enzyme'

import 'jest-styled-components'

import Component from '../index'

describe('TrackMap', () => {
  const defaultProps = {
    initialRegion: {
      lat: 13.32124,
      lon: 31.32143
    }
  }

  it('render snapshot', () => {
    const wrapper = shallow(<Component {...defaultProps} />)

    expect(wrapper).toMatchSnapshot()
  })
})
