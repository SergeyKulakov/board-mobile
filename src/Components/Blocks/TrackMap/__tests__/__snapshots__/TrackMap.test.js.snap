// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TrackMap render snapshot 1`] = `
<Styled(MapView)
  initialRegion={
    Object {
      "latitude": 13.32124,
      "latitudeDelta": 0.1,
      "longitude": 31.32143,
      "longitudeDelta": 0.1,
    }
  }
  mapType="standard"
  provider="google"
  showsUserLocation={true}
  zoomEnabled={true}
/>
`;
